src/Common/ImplementsApiInheritDocRule.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace ArchitectureSniffer\Common;
+
+use ArchitectureSniffer\SprykerAbstractRule;
+use PHPMD\AbstractNode;
+use PHPMD\Node\MethodNode;
+use PHPMD\Rule\ClassAware;
+
+class ImplementsApiInheritDocRule extends SprykerAbstractRule implements ClassAware
+{
+    protected const RULE = 'Every API public method must also contain the {@inheritDoc} tag in docblock.';
+
+    /**
+     * @var string[]
+     */
+    protected $apiClasses = ['Facade', 'QueryContainer', 'Client', 'Service', 'Plugin'];
+
+    /**
+     * @return string
+     */
+    public function getDescription()
+    {
+        return static::RULE;
+    }
+
+    /**
+     * @param \PHPMD\AbstractNode $classNode
+     *
+     * @return void
+     */
+    public function apply(AbstractNode $classNode)
+    {
+        $classNamespace = $classNode->getNamespaceName();
+        if (strpos($classNamespace, 'Dependency\\') !== false) {
+            return;
+        }
+
+        $className = $classNode->getName();
+        if (!preg_match('/(' . implode('|', $this->apiClasses) . ')$/', $className)) {
+            return;
+        }
+
+        /** @var \PHPMD\Node\InterfaceNode $classNode */
```

Comment:
Is this valid? I think you want multiline docblock as single one here.

Reply:
no, i don't

Reply:
why?

Reply:
done